inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -799,6 +801,48 @@ private KBHandle update(KnowledgeBase kb, UpdateAction aAction)
         return handles;
     }
 
+    @Override
+    public KBStatement getExistingStatement(KnowledgeBase kb, KBHandle aInstance,
+        KBHandle aProperty, String aObject)
+    {
+        if (existThisStatement(kb, aInstance, aProperty, aObject)) {
+            KBStatement kbStatement = new KBStatement();
+            kbStatement.setInstance(aInstance);
+            kbStatement.setProperty(aProperty);
+            kbStatement.setValue(aObject);
+            return kbStatement;
+        }
+
+        return null;
```

Comment:
Use `Optional` if possible

Reply:
The code is used for writing statements directly in KB. I have removed it.